xapian-core/include/xapian/weight.h

```diff
@@ -435,8 +449,9 @@ class XAPIAN_VISIBILITY_DEFAULT TfIdfWeight : public Weight {
      *     @li 'b': Boolean    wdfn=1 if term in document else wdfn=0
      *     @li 's': Square     wdfn=wdf*wdf
      *     @li 'l': Logarithmic wdfn=1+log<sub>e</sub>(wdf)
+     *     @li 'm': Max_wdf     wdfn=wdf/max_wdf
      *
-     *     The Max-wdf and Augmented Max wdf normalizations haven't yet been
+     *     The Max_wdf and Augmented Max_wdf normalizations haven't yet been
```

Comment:
Doesn't this PR implement `Max_wdf`? In which case it's only the augmented version that needs mentioning here now.